src/adv.py

```diff
@@ -33,19 +42,79 @@
 room['narrow'].n_to = room['treasure']
 room['treasure'].s_to = room['narrow']
 
+# Link items to rooms
+room['outside'].items = [items['sword']]
```

Comment:
Excellent breakdown of linking those items to the rooms, just as the repo has rooms linked to each other!